webapp/src/main/webapp/src/app/components/confirm-delete/confirm-delete.component.ts

```diff
@@ -13,28 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, Output, EventEmitter, ViewChildren, QueryList } from "@angular/core";
+import { Component, Output, EventEmitter, ViewChildren, QueryList } from '@angular/core';
 
-import { ModalDirective } from "ngx-bootstrap/modal";
+import { ModalDirective } from 'ngx-bootstrap/modal';
 
 @Component({
-  selector: "confirm-delete-dialog",
-  templateUrl: "confirm-delete.component.html"
+  selector: 'app-confirm-delete-dialog',
```

Comment:
It's a common practice to prevent name collisions.
Based on angular [style guides "Directive custom prefix"](https://angular.io/guide/styleguide#style-02-08)

By default it's "app-" but we can specify any other. What is your preference? What about using "mrx-"?

Reply:
That's great! I was not aware of this convention.